pkg/e2e/operators/cloudingress/installed.go

```diff
@@ -2,32 +2,96 @@ package cloudingress
 
 import (
 	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
 
 	"github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
 	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/spf13/viper"
 
+	"github.com/openshift/osde2e/pkg/common/config"
+	appsv1 "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // tests
+
 var _ = ginkgo.Describe(CloudIngressTestName, func() {
+
+	var defaultDesiredReplicas int32 = 1
+
 	h := helper.New()
 
-	ginkgo.It("is a placeholder", func() {
-		_, _ = h.Kube().CoreV1().Pods(CloudIngressNamespace).Get(context.TODO(), "something", metav1.GetOptions{})
+	checkDeployment(h, CloudIngressNamespace, OperatorName, defaultDesiredReplicas)
+})
 
-		tests := []struct {
-			Name string
-		}{}
+func checkDeployment(h *helper.H, namespace string, name string, defaultDesiredReplicas int32) {
+	// Check that the operator deployment exists in the operator namespace
+	ginkgo.Context("deployment", func() {
+		ginkgo.It("should exist", func() {
+			deployment, err := pollDeployment(h, namespace, name)
+			Expect(err).ToNot(HaveOccurred(), "failed fetching deployment")
+			Expect(deployment).NotTo(BeNil(), "deployment is nil")
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
```

Comment:
```suggestion
		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
		
```

let's put a newline here to make the tests more clearly separate